torch/linalg/__init__.py

```diff
@@ -923,13 +923,15 @@
 """)
 
 matrix_rank = _add_docstr(_linalg.linalg_matrix_rank, r"""
-matrix_rank(A, tol=None, hermitian=False, *, out=None) -> Tensor
+linalg.matrix_rank(A, *, atol=None, rtol=None, hermitian=False, out=None) -> Tensor
+linalg.matrix_rank(A, tol, hermitian=False, *, out=None) -> Tensor
 
 Computes the numerical rank of a matrix.
 
 The matrix rank is computed as the number of singular values
 (or eigenvalues in absolute value when :attr:`hermitian`\ `= True`)
-that are greater than the specified :attr:`tol` threshold.
+that are greater than `max(atol, σ₁*rtol)` threshold,
```

Comment:
We settled for not using unicode because of rendering issues. This should be in LaTeX (see also the next line)